.github/workflows/nodejs.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     steps:
       - uses: rokroskar/workflow-run-cleanup-action@master
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ github.token }}
```

Comment:
see https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context for detail.

Reply:
is there any real difference?

>A token to authenticate on behalf of the GitHub App installed on your repository. This is functionally equivalent to the `GITHUB_TOKEN` secret.